vagrant_support/cron.yml

```diff
@@ -0,0 +1,45 @@
+- name: set cron mailto
+  cron:
+    env: yes
+    name: MAILTO
+    value: vagrant@localhost
+
+- name: cron remove-idle-group-members
+  cron:
+    hour: 7
+    minute: 0
+    job: 'cd /vagrant; ./scripts/remove_idle_group_members.pl'
```

Comment:
Hope that it shouldn't fail :). Though what @floatingatoll said, should be considered.